usagov/vote-gov-drupal
# VOTE- 3134 replace online and in-person date fields

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-3134

## Description

Replace double fields with default date fields for online and in person.
Delete the double fields from the system.

## Deployment and testing

### Post-deploy steps

1. lando retune

### QA/Testing instructions

1. Visit http://vote-gov.lndo.site/node/7/edit
2. Set the in-person and online deadline fields.
3. Confirm the specific deadlines display correctly on the page.
4. Confirm the dates translate when switching languages.
5. Clear the specific deadlines and confirm the default deadlines display on the page.

## Checklist for the Developer




## Checklist for the Peer Reviewers
